src/App.jsx

```diff
@@ -1,16 +1,29 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { registerRootComponent } from 'expo';
+import * as Font from 'expo-font';
 import { Provider } from 'react-redux';
 import { createAppContainer } from 'react-navigation';
 import AppNavigator from './Navigator';
 import store from './redux/store';
 
 const AppContainer = createAppContainer(AppNavigator);
 
-const App = () => (
-  <Provider store={store}>
-    <AppContainer />
-  </Provider>
-);
+
+export default class App extends React.Component {
+  async componentDidMount() {
```

Comment:
You can solve this with a functional component, I trust you ;)

Reply:
fixed